src/llmcompressor/transformers/finetune/README.md

```diff
@@ -159,12 +159,12 @@ apply(
     model_name_or_path=model,
     dataset_name=dataset_name,
     run_stages=run_stages,
-    output_dir=output_dir,
```

Comment:
Why not keep this here if it's still supported?

Reply:
yes you are right